src/test/java/com/nerdapplabs/test/Test_Logout.java

```diff
@@ -0,0 +1,31 @@
+package com.nerdapplabs.test;
+
+import java.io.IOException;
+
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+
+public class Test_Logout extends Test_Login {
```

Comment:
This is wrong to extend one test case to other. It means one is dependent to other. Test are always work as a single(separate) unit. No dependency to other test. Understand and fix it.